### PR TITLE
Set force=true when removing a windows directory

### PR DESCRIPTION
**Pull request by michelsm**
### What does this PR do?

Provides an additional check when file.absent is being applied to a directory. This allows windows directories to be forcefully removed by setting the force field equal to true.

This PR fixes an inconsistency that currently exsists when using file.absent to remove a single file on windows vs removing a windows directory.

Currently removing windows...
- file uses file.remove with force=true
- directory uses file.remove with force=false

As a result, this PR will ensure that the removal of windows directories is executed with the same logic.

### What issues does this PR fix or reference?

Inability for users to forcefully remove windows directories (becomes an issue with permissions).
Inconsistencies between the removal of windows files and windows directories with file.absent.

### Previous Behavior

Prior to this PR, the functionality of forceful removal was not implemented for directories.
This poses an issue for windows users due to permission constraints.

### New Behavior

Windows users have the ability to forcefully remove directories without the interference of permissions. The changes implemented for directories now follow the same logic that was previously set for files.

### Tests written?

No

### CODE REVIEW

**Review by twangboy — APPROVED** [review]

**Comment by michelsm** [comment]
Since this PR has received approval, when can I expect these changes to be merged?

File: salt/states/file.py [hunk 1]
```diff
@@ -1453,7 +1453,10 @@ def absent(name):
             ret['comment'] = 'Directory {0} is set for removal'.format(name)
             return ret
         try:
-            __salt__['file.remove'](name)
+        	if salt.utils.is_windows():
```
Comment (twangboy) [inline salt/states/file.py]: Needs another space in the indentation here... It's failing lint.

**Review by twangboy — CHANGES_REQUESTED** [review]

**Comment by michelsm** [comment]
Spacing has been fixed.

**Review by twangboy — APPROVED** [review]

**Comment by michelsm** [comment]
Hi, I'm new to this and can see that this PR has been approved. Is there anything else I need to do to get my PR merged?

**Comment by rallytime** [comment]
Hi @michelsm and welcome!

I have assigned @terminalmage to take a look at this. Once he has a chance to review it, we can go from there. Thank you for your contribution!

**Review by terminalmage — APPROVED** [review]

**Comment by terminalmage** [comment]
@michelsm Thanks! And congrats on your first Salt PR!

**Pull request merged**